apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -48,6 +48,13 @@ const ProgressBubbleSet = React.createClass({
     disabled: PropTypes.bool.isRequired
   },
 
+  getIcon(level) {
+    if (level.icon === null){
```

Comment:
I dont know if we ever hit this in real code, but in storybook it looks like there are instances where level.icon is undefined rather than null (and thus level.icon.substring throws an exception). We can probably just make line 52: 'if (level.icon) {`